packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -0,0 +1,83 @@
+const algoliasearch = require(`algoliasearch`)
+const crypto = require(`crypto`)
+
+const client = algoliasearch(`OFCNCOG2CU`, `f54e21fa3a2a0160595bb058179bfb1e`)
+var index = client.initIndex(`npm-search`)
+
+const createContentDigest = obj =>
+  crypto
+    .createHash(`md5`)
+    .update(JSON.stringify(obj))
+    .digest(`hex`)
+
+exports.sourceNodes = async (
+  { boundActionCreators, createNodeId },
+  { keywords }
+) => {
+  const { createNode } = boundActionCreators
+
+  console.log(`Grabbing local NPM packages...`)
+
+  let buildFilter = []
+
+  keywords.forEach(keyword => {
+    buildFilter.push(`keywords:${keyword}`)
+  })
+
+  const data = await index.search({
+    query: ``,
+    filters: `(${buildFilter.join(` OR `)})`,
+    hitsPerPage: 1000,
+  })
+
+  // TODO: Currently the gatsby-transformer-remark plugin
+  // has a hard time with markdown that includes
+  // images -> for now this filters out those packages
+
+  data.hits.forEach(hit => {
+    if (hit.readme.includes(`![`)) {
+      return
+    }
+
+    const parentId = createNodeId(`plugin ${hit.objectID}`)
+    console.log(parentId)
+    const readmeNode = {
+      id: createNodeId(`readme ${hit.objectID}`),
+      parent: parentId,
+      slug: `/packages/en/${hit.objectID}`,
+      children: [],
+      internal: {
+        type: `NPMPackageReadme`,
+        mediaType: `text/markdown`,
+        content: hit.readme,
```

Comment:
To fix the `loadNodeContent` error — check that hit.readme is always defined as a string — so check if it's not defined and if it isn't, set it as any empty string.